docs/FAQ.md

```diff
@@ -1,11 +1,159 @@
+[skinit-spec]: https://www.amd.com/system/files/TechDocs/24593.pdf
+
+### So, why TrenchBoot?
+
+- Managing multiple independent components individually can be confusing
+- Most DRTM systems are platform-dependent
+- Most DRTM systems can only measure entities already present in RAM at the
+    time DRTM sequence is initiated
+- Most DRTM systems support only one attestation action
+
+## 2. How does TrenchBoot work? (trenchboot architecture)
+
+### Architecture Overview
+
+*The full architecture overview can be found on
+[the corresponding page](./theory/Architecture.md).*
+![Overview](img/Architectural_Flow.png)
+This diagram shows a general overview of the overall architecture of a
+TrenchBoot system. In Bootstrap phase the system environment is measured and
+that information stored in the dynamic PCRs, using whatever method is
+appropriate for the given hardware. The intermediate phase analyzes the
+information collected in the bootstrap phase to decide if it wants to continue
+booting and pass control to the target OS, or if it wants to stop immediately.
+If desired, additional evidence/measurements can be gathered in this phase. Once
+ the integrity of the system is verified the TrenchBoot kernel passes off
+ control to the actual desired OS to initiate the runtime phase.
+
+### Bootstrap Phase - GRUB
+
+Typically, when attempting to establish a DRTM, TBoot is loaded in order to
+carry out TXT on Intel, or SKINIT is used on AMD. TrenchBoot instead utilizes
+its own system, which is a custom version of GRUB that has commands built in to
+carry out a TrenchBoot "Secure Launch". This customized GRUB introduces the
+`slaunch` and `slaunch_module` commands. A working example of the TrenchBoot
+GRUB can be found at
+[https://github.com/3mdeb/meta-trenchboot](
+    https://github.com/3mdeb/meta-trenchboot).
+The custom GRUB works with Linux and Multiboot2 boot protocols for both Intel
+and AMD devices, but only in case of legacy boot, without support for UEFI
+(although it's available on a separate branch).
+
+### Intermediate Phase - Linux Kernel / U-ROOT (Initramfs)
+
+The intermediate phase processes information collected in the bootstrap phase
+and converts it into normalized data. This data is eventually used to determine
+whether to boot into the target OS. The kernel and u-root initramfs
+together make up Trenchboot's "Security Engine", which is an intermediate
+mini-OS that serves to process data gathered by that bootstrap phase. During the
+intermediate phase, the kernel and initramfs work together to measure block
+devices, individual files, SMBUS/DMI information and provides unseal based,
+external device and network based attestation. Trenchboot calls this
+functionality secure launch (aka slaunch). The table in section 4 lists the
+specific repo and branch to use when building the kernel or u-root.
+
+## 3. How to use Trenchboot? (practical usage)
+
+### Trenchboot GRUB TXT Config with Serial Communication
```

Comment:
This seems out of place and incomplete.

Also, there is a link to user documentation [on the front page](https://github.com/TrenchBoot/TrenchBoot.github.io/blob/cleanup/docs/index.md?plain=1#L17), I was hoping that it would be enough to avoid questions like this one.

Reply:
https://github.com/TrenchBoot/TrenchBoot.github.io/pull/41/commits/1fe22a330c4672ea2ecaf9a90a9071edb1386b6a

Reply:
Broken links, numbers after `#` were not changed on the list at the top.

Reply:
https://github.com/TrenchBoot/TrenchBoot.github.io/compare/25021ec141d2a88fe039a75c8743ba4dd733f2e3..3c9d6af92c149d463430036cddd65a3ba2e58968